### DISCUSSION TITLE
Status? Thoughts?

### DISCUSSION THREAD

**Discussion by jmatsushita** [pmndrs/react-three-jolt#7] [General]
Hi there,

I'm wondering if there's still an intention to pursue the integration of r3f with jolt? AFAICT it's [possible to integrate jolt and threejs](https://github.com/jrouwe/JoltPhysics.js/blob/d8339196c5f461c0ec8697d216ecec92c1e706f1/Examples/js/example.js). I'm curious about what your experience was attempting to write react-three-jolt? Any obstacles you ran into or thoughts you might be able to share?

Cheers,

Jun

**Comment by jmatsushita** [comment]
FYI, there is this sandbox which integrates jolt with r3f https://codesandbox.io/p/devbox/jolt-physics-react-three-fiber-demo-hxkkje?file=%2Fsrc%2FApp.tsx

**Comment by isaac-mason** [comment]
There will be more activity here soon 👀

**Comment by isaac-mason** [comment]
To elaborate -

This repository has been acting as a placeholder for a comprehensive r3f jolt library.

I created that sandbox as a demonstration of what a very simple react three fiber + jolt integration looks like, but couldn't commit to building a full fledged rt/jolt library.

Using jolt directly is very doable, people are already using JoltPhysics.js with threejs and other libraries. Much of the value of creating a react-three/jolt library is ease of use and interop with the rest of the react-three ecosystem.

More recently @DennisSmolek has been working on a react-three/jolt library, so this repository won't be placeholder for much longer. Stay tuned 👀
